src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './Dropdown.css';
+// We use several `Button` classes and CSS variables.
+import '../Button.css';
+
+interface DropdownProps {
+  /**
+   * Children are expected to be a `DropdownTrigger` and `DropdownMenu`,
+   * although this is not enforced.
+   */
+  children: React.ReactNode;
+  /**
+   * Should the menu be aligned to the left or right edge of its toggle?
+   */
+  align?: 'left' | 'right';
+  className?: string;
+}
+
+/**
+ * Accessible, JavaScript-free dropdowns using the `<details>` element.
```

Comment:
:tada: but also this is rendered with JavaScript so :upside_down_face: . I'm still a fan, just enjoying the irony!

Reply:
😝